src/pages/younghyunHan/Main/Main.js

```diff
@@ -1,8 +1,158 @@
 import './Main.scss';
+import Nav from '../../../components/Nav/Nav';
 import '../../../styles/variables.scss';
+import Article from './Article';
+import { useState, useEffect } from 'react';
 
 function Main() {
-  return <></>;
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/younghyunHan/articleData.json')
+      .then(res => res.json())
+      .then(data => {
+        setArticles(data);
+      });
+  }, []);
+
+  return (
+    <div className="main">
+      <Nav />
+      <main>
+        <div className="feeds">
+          {articles.map((content, index) => {
+            return (
+              <div key={index}>
+                <Article
+                  userName={content.userName}
+                  feedImg={content.feedImg}
+                  userMan={content.userMan}
+                />
+              </div>
+            );
+          })}
+        </div>
+        <div className="mainRight">
```

Comment:
mainRight도 컴포넌트로 만들어주면 좋을 것 같아요
이렇게 해주면 main.js 내용, scss 내용도 더 간단해질 것 같음!
```suggestion
        <MainRight />
```